package.json

```diff
@@ -19,9 +19,13 @@
     "detect-browser": "^2.1.0",
     "event-stream": "^3.3.4",
     "execa": "^0.9.0",
-    "gulp": "^3.9.1",
+    "gulp": "github:gulpjs/gulp#4.0",
```

Comment:
Can we somehow get rid of this change? I won't be able to use my globally installed gulp@3.9.1 for SweetAlert2,

Reply:
I was pretty sure that the globally installed gulp cli delegates to the project's locally installed gulp

Did you try it? If not I'll test it out

Reply:
This is what's happening in my env when I try to run any gulp task:

~[](https://i.imgur.com/mLNLMFM.png)

Reply:
![](https://i.imgur.com/r03JelX.png)

Reply:
Ok. Why do you need to use your globally installed gulp?

Reply:
btw. wtf is happening with gulp? why 4.0.0 isn't installable? https://github.com/gulpjs/gulp/releases/tag/v4.0.0

Reply:
We could add a package script `"gulp": "gulp"` so you can pass arguments in through `npm`, e.g. `npm run gulp -- build:styles`.

Reply:
> Ok. Why do you need to use your globally installed gulp?

Good question :) I don't need it. Resolved.